### PR TITLE
GCCDEV-5057: PHP 8.3//Laravel 11 support.

### PR DESCRIPTION
**Pull request by davidxhill**

### CODE REVIEW

File: .phpactor.json
Comment (rmarganti) [inline .phpactor.json]: you know the drill

```suggestion
```

File: composer.json
Comment (rmarganti) [inline composer.json:17]: I wonder when/if `laravel/doctrine` is going to move on to doctrime/orm:3

**Review by rmarganti — CHANGES_REQUESTED** [review]
Looks good pending stable versions

File: composer.json
Comment (davidxhill) [reply composer.json:17]: they market themselves as `A drop-in Doctrine ORM 2 implementation for Laravel 5+ and Lumen ` So i am going to assume never.. I am just happy it gets updated at this point.

File: .phpactor.json
Comment (rmarganti) [inline .phpactor.json]: As mentioned elsewhere, I think we should `.gitignore` these.

File: composer.json
Comment (rmarganti) [inline composer.json]: Request got merged. Just waiting on response.

**Review by rmarganti — CHANGES_REQUESTED** [review]
Feel free to make the changes you need to and self-approve/merge when ready.

**Pull request merged**
